frontend/src/metabase/nav/components/AppBar/AppBarLarge.styled.tsx

```diff
@@ -21,7 +21,7 @@ export const AppBarRoot = styled.div<AppBarRootProps>`
 `;
 
 export interface AppBarLeftContainerProps {
-  isNavBarVisible?: boolean;
+  isNavBarEnabled?: boolean;
```

Comment:
Rename the boolean to reflect its behavior.
`isNavBarVisible` means whether or not the navbar is on the screen where as `isNavBarEnabled` means we enabled the navbar but it could be closed.